### PR TITLE
build: fix chromedriver build issue

### PR DESCRIPTION
**Pull request by blaggacao**
# Context

Is failing because of this https://github.com/input-output-hk/cardano-js-sdk/pull/756

# Proposed Solution

# Important Changes Introduced

### CODE REVIEW

**Review by rhyslbw — APPROVED** [review]

**Comment by blaggacao** [comment]
@rhyslbw will fold this (+ proof of build) into https://github.com/input-output-hk/cardano-js-sdk/pull/756

**Comment by gytis-ivaskevicius** [comment]
Let's merge this since #756 is not ready yet

**Comment by gytis-ivaskevicius** [comment]
Ready to be merged :))

**Review by rhyslbw — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -72,13 +72,14 @@
...
         buildPhase = ''
-          yarn build
+          yarn workspace @cardano-sdk/cardano-services run build
         '';
         # override installPhase to only install what's necessary
         installPhase = ''
           runHook preInstall
 
           mkdir -p $out/libexec/$sourceRoot $out/bin
+          yarn workspace @cardano-sdk/cardano-services-client run build:version
```
Comment (mkazlauskas) [inline flake.nix:82]: Shouldn't this be in `buildPhase`? :thinking:

**Review by mkazlauskas — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (gytis-ivaskevicius) [reply flake.nix:82]: I was getting some read-only write error, not sure why. It works here thus I did not bother looking into it

**Pull request merged**